docs/modules/ROOT/pages/microprofile-certification/5.36.0/README.adoc

```diff
@@ -0,0 +1,9 @@
+= MicroProfile 4.1 Certification Summaries
+
+Payara Server is certified https://projects.eclipse.org/projects/technology.microprofile/[MicroProfile 4.1] implementation.
+As proof of compatibility, output of Jakarta EE Compatibility Test Suite tests are presented for each certified version.
```

Comment:
`output of Jakarta EE Compatibility Test Suite`

Must be output of MicroProfile TCK Test Suite.